Default (OSX).sublime-mousemap

```diff
@@ -0,0 +1,12 @@
+[
+    {
+        "button": "button1", 
+        "modifiers": ["ctrl"], 
```

Comment:
This should probably be `["alt"]`. `ctrl+button1` is used for right clicks on some older Macs (where there is usually only one button anyway).